pontoon/base/models.py

```diff
@@ -1749,22 +1749,21 @@ def get_filtered_entities(self, locale, query, rule, match_all=True):
             # so that the first one for each plural form will be the active one.
             plural_candidates = (
                 self
+                .filter(
+                    resource__translatedresources__locale=locale,
+                    obsolete=False
+                )
```

Comment:
This makes things slow (e.g. slower than current master when filtering Firefox/All Resources).

If you revert 3f668c7 instead, we're good.